core/store/ledgerstore/ledger_store.go

```diff
@@ -187,7 +187,7 @@ func (this *LedgerStoreImp) init() error {
 
 func (this *LedgerStoreImp) initCurrentBlock() error {
 	currentBlockHash, currentBlockHeight, err := this.blockStore.GetCurrentBlock()
-	if err != nil {
+	if err != nil && err != scom.ErrNotFound {
```

Comment:
this seems unsound, not found is beyond our expectation if the db is initialized properly. so ignore it and silently set `this.currBlockHash = HASH_EMPTY` and continue run will have problem.

Reply:
The main reason is the first time get the value will return  not found,